server/pulp/server/webservices/views/repositories.py

```diff
@@ -0,0 +1,1214 @@
+        """
+        Create a new scheduled publish.
+
+        :param request: WSGI request object
+        :type  request: django.core.handlers.wsgi.WSGIRequest
+        :param repo_id: id of the repository
+        :type  repo_id: str
+        :param distributor_id: id of the distributor
+        :type  distributor_id: str
+
+        :return: Response containing a dict for the new scheduled publish
+        :rtype : django.http.HttpResponse
+        :raises: pulp_exceptions.UnsupportedValue if unsupported fields are included in body
+        """
+
+        manager = manager_factory.repo_publish_schedule_manager()
+        publish_options = {'override_config': request.body_as_json.pop('override_config', {})}
+        schedule = request.body_as_json.pop('schedule', None)
+        failure_threshold = request.body_as_json.pop('failure_threshold', None)
+        enabled = request.body_as_json.pop('enabled', True)
+        if request.body_as_json:
+            raise pulp_exceptions.UnsupportedValue(request.body_as_json.keys())
+
+        schedule = manager.create(repo_id, distributor_id, publish_options,
+                                  schedule, failure_threshold, enabled)
+        ret = schedule.for_display()
+        ret['_href'] = reverse('repo_publish_schedule_resource', kwargs={
+            'repo_id': repo_id, 'distributor_id': distributor_id, 'schedule_id': schedule.id
+        })
+        response = generate_json_response_with_pulp_encoder(ret)
+        return generate_redirect_response(response, ret['_href'])
+
+
+class RepoPublishScheduleResourceView(ScheduleResource):
+    """
+    View for a single scheduled publish.
+    """
+
+    def __init__(self):
+        """
+        Initialize RepoPublishScheduleResourceView.
+        """
+
+        super(RepoPublishScheduleResourceView, self).__init__()
+        self.manager = manager_factory.repo_publish_schedule_manager()
+
+    @auth_required(authorization.READ)
+    def get(self, request, repo_id, distributor_id, schedule_id):
+        """
+        Retrieve information about a scheduled publish.
+
+        :param request: WSGI request object
+        :type  request: django.core.handlers.wsgi.WSGIRequest
+        :param repo_id: id of the repository
+        :type  repo_id: str
+        :param distributor_id: id of the distributor
+        :type  distributor_id: str
+        :param schedule_id: id of the scheduled publish to be retrieved
+        :type  schedule_id: str
+
+        :return: Response containing serialized dict for the scheduled publish
+        :rtype : django.http.HttpResponse
+        """
+
+        self.manager.validate_distributor(repo_id, distributor_id)
+        resource_href = reverse('repo_publish_schedule_resource', kwargs={
+            'repo_id': repo_id, 'distributor_id': distributor_id, 'schedule_id': schedule_id
+        })
+        return self._get(schedule_id, resource_href)
+
+    @auth_required(authorization.UPDATE)
+    @json_body_required
```

Comment:
from the docs i can see that all params are optional, therefore we should use here @json_body_allow_empty
Same like for scheduled sync update.